app/src/main/java/com/example/trempelapp/presentation_layer/logInScreen/TrempelLogInActivity.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.trempelapp.presentation_layer.logInScreen
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.lifecycle.ViewModelProvider
+import com.example.trempelapp.BaseActivity
+import com.example.trempelapp.TrempelApplication
+import com.example.trempelapp.databinding.ActivityLogInBinding
+import com.example.trempelapp.presentation_layer.homeScreen.HomeScreenActivity
+
+class TrempelLogInActivity : BaseActivity() {
+
+    private val trempelLogInViewModel by lazy {
+        ViewModelProvider(this, TrempelLogInViewModelFactory())[TrempelLogInViewModel::class.java]
+    }
+
+    private val binding by lazy {
+        ActivityLogInBinding.inflate(layoutInflater)
+    }
+
+    override fun injectDagger() {
+        trempelLogInViewModel.injectDagger(application as TrempelApplication)
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        supportActionBar?.hide()
+        setContentView(binding.root)
+        injectDagger()
```

Comment:
you will call it twice. You are already calling injectDagger in your BaseViewModel

Reply:
done